frontend_vue/src/composables/useTokens.ts

```diff
@@ -0,0 +1,96 @@
+import { TOKENS_TYPE } from '@/components/constants.ts';
+import { ref } from 'vue';
+import generateManagedToken from '@/components/tokens/my_sql/generateManagedToken';
+
+type TokenOperationType = {
+  label: string;
+  description: string;
+  documentationLink: string;
+  icon: string;
+  instruction: string;
+  getNewTokenData?: (data: any) => void;
+  getManageTokenData?: (data: any) => void;
+};
+
+type TokenOperationsType = {
+  [key: string]: TokenOperationType;
+};
+
+export function useTokens() {
```

Comment:
is there a reason to define this as a function? can't we just export `tokenOperations`?